_pages/about-us/teams/acqstack.md

```diff
@@ -1,48 +1,118 @@
-- **Work in cross-functional teams.** Including diverse expertise in the process from the very beginning is key to a successful acquisition. That’s why we work in cross-functional teams representing design, technical, product and procurement expertise from day one on each of our projects, and encourage our partners to do the same.
-- **Focus on the user need.** We believe that happy users are the sign of great digital services, so we engage with end-users early and often to make sure that our procurements and products address their needs.
-- **Treat all parties in the process with empathy.** We work collaboratively with our agency partners, vendors, and the rest of TTS to achieve our mission. As such, we approach each of these stakeholders with the same empathy and open-mindedness as we do the end-users of the products and services we procure.
-- **Work in the open.** We believe transparency, accountability and collaboration ultimately result in better-quality products and services. We are committed to opening up our process and our source code as much a possible, and encourage our partners and vendors to do the same.
-- **Continuously improve.** Our work is never done - there is always room to improve. We conduct regular retrospectives on our projects, with our team, and with our partners and vendors to continuously improve our methods and practices over time.
+Members of the Acquisition team come from all over: inside GSA, consulting firms, peace corps, and other government agencies. Collectively, we work as the acquisition part of the overall 18F cross-functional team bringing our expertise to help agency partners (experts in their mission) produce more effective procurement strategies, implement modern procurement practices, and better vendor outcomes. We’re experienced in a wide variety of procurements and have tons of stories of procurements that have gone well and some that haven’t.  Ask us!
 
+### Our Vision & Mission
 
-## What We Do
-We focus on procurements of intentionally modest scope for open-source digital products and services, which we procure and deliver incrementally, so that we can assess the results more quickly, and refine our approaches over time.
+The Acquisition Chapter exists to prove that government acquisitions can be joyful for our employees, agency partners, and industry partners. We do this, by ensuring that informed buyers can easily and confidently access the right products and services to meet their agency’s  mission. No matter what the agency partner is undertaking or trying to solve, the acquisition process will be unavoidable and it can be used as an amazing point for leverage to improve overall outcomes.
+
+As a team, we are dedicated to bringing an agile, human centered, and cross-functional approach to procurements that will better prepare our partners to work with modern vendor teams iteratively; developing open source software products to help agencies achieve their mission.
+
+### Our Value Proposition
```

Comment:
I'd double-check with the content team, but I think we use title case in headings. @jenniferthibault can you confirm?

Reply:
I'm not sure if the handbook is following a specific content style guide, but I'd wager it sticks to the content team's guide which recommends sentence case (Is this what you're going for, Tim? I have to google these every time myself to make sure I'm using the right label!) https://content-guide.18f.gov/capitalization/#headings

Sentence case uses `The title of a section heading` instead of `The Title of a Section Heading`

Reply:
Thanks Jen. Yes, sentence case was what I meant. Brain only partially engaged. Needs coffee.